src/config.rs

```diff
@@ -74,6 +74,13 @@ pub enum Config {
     #[strum(props(default = "0"))]
     SentboxMove, // If `MvboxMove` is true, this config is ignored. Currently only used in tests.
 
+    /// Watch for new messages in the "Mvbox" (aka DeltaChat folder) only.
+    ///
+    /// This will not entirely disable other folders, e.g. the spam folder will also still
+    /// be watched for new messages.
```

Comment:
> and Inbox should be watched for replies to DC messages sent from classic MUA.

No, the use case (esp. webratte's use case - unfortunately the whole thread in the forum is in German :/ ) is that people are looking at their inbox with another MUA and really only want to show messages from the DeltaChat folder in their MUA. In former versions of DC, they disabled `inbox_watch`.

Reply:
SIEVE filter will not move replies to DC messages to DC folder, so you still need to watch it in case a reply to DC message goes there?

I don't get the use case then.

Reply:
Why isn't it possible to just enable "show only DC messages"?

Reply:
> SIEVE filter will not move replies to DC messages to DC folder, so you still need to watch it in case a reply to DC message goes there?

Webratte is using an alias address `address+deltachat@mailbox.org`, and using a filter to move all emails that went to this address to the DeltaChat folder. He wants to show exactly the emails that went to `address+deltachat@mailbox.org` in DC, and show all other emails in K9 (which ignores the DeltaChat folder).